src/vm.rs

```diff
@@ -547,8 +547,17 @@ impl<T: GuestStagePageTable> Vm<T, VmStateFinalized> {
                             }
                         };
 
-                        // We only emulate WFI for now. Everything else gets redirected as an illegal instruction exception.
+                        // We only emulate WFI and Crrss for PMU registers for now.
+                        // Everything else gets redirected as an illegal instruction exception.
                         match inst.instruction() {
```

Comment:
I think this was mentioned in a previous iteration, but it's maybe time for instruction emulation handling to go in its own method.